### PR TITLE
Integrate swagger with authorization

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Integrate swagger with authorization token
PR

### CODE REVIEW

File: Doppler.Currency/Startup.cs [hunk 1]
```diff
@@ -1,10 +1,3 @@
-using System;
-using System.Collections.Generic;
-using System.Diagnostics.CodeAnalysis;
-using System.Linq;
-using System.Net;
-using System.Net.Http;
-using System.Security.Authentication;
```
Comment (andresmoschini) [inline Doppler.Currency/Startup.cs:7]: This change seems to not be related with the scope of the task.

File: Doppler.Currency/Startup.cs [hunk 2]
```diff
@@ -63,6 +63,35 @@ public void ConfigureServices(IServiceCollection services)
                     Version = "v1",
                     Description = "API for Doppler Currency"
                 });
+
+                var security = new Dictionary<string, IEnumerable<string>>
+                {
+                    {"Bearer", new string[] { }},
```
Comment (andresmoschini) [inline Doppler.Currency/Startup.cs:69]:
```suggestion
                    { "Bearer", new string[] { } },
```

File: Doppler.Currency/Startup.cs [hunk 3]
```diff
@@ -63,6 +63,35 @@ public void ConfigureServices(IServiceCollection services)
...
+                    {
+                        new OpenApiSecurityScheme
+                        {
+                            Reference = new OpenApiReference
+                            {
+                                Type = ReferenceType.SecurityScheme,
+                                Id = "Bearer"
+                            }
+                        },
+                        Array.Empty<string>()
```
Comment (andresmoschini) [inline Doppler.Currency/Startup.cs:92]: In one place you are using `new string[] { }` and in another place you are using `Array.Empty<string>()` to create a empty string array. Is it possible choose one option?

**Comment by andresmoschini** [comment]
Great, could you add a screenshot to the PR to understand what is the effect of this change?

File: Doppler.Currency/Startup.cs (hunk 3 above)
Comment (jhoffman-ms) [reply Doppler.Currency/Startup.cs:92]: I used the Array.Empty<string>() option for both cases

**Comment by jhoffman-ms** [comment]
I will add a screenshot

**Comment by andresmoschini** [comment]
🚀

**Pull request merged**